lib/http/timeout/null.rb

```diff
@@ -46,6 +46,28 @@ def write(data)
         @socket << data
       end
 
+      # Retry reading
+      def __rescue_readable(&block)
```

Comment:
If I recall, it's better to not explicitly reference `&block` since it's less performant than implicit. @tarcieri do you remember? I can't find the article on it anymore.